extensions/fairifier-rdf-extension/scripts/rdf-skeleton-list.js

```diff
@@ -25,9 +27,8 @@ RdfSkeletonListDialog.prototype._constructFooter = function(footer) {
     var self = this;
     
     $('<button></button>').addClass('button').html("&nbsp;&nbsp;Load&nbsp;&nbsp;").click(function() {
-        $.post("command/rdf-extension/load-rdf-skeleton",  {projectId : listData[$( "select :selected" ).attr('id')].project},function(data){
+        $.post("command/rdf-extension/load-rdf-skeleton",  {project: self._project,schema: JSON.stringify(self._schema) ,projectId : listData[$( "select :selected" ).attr('id')].project},function(data){
```

Comment:
This can be made a lot more readable by introducing some newlines and indentation.